packages/api/test/e2e/ethBridge.e2e.ts

```diff
@@ -165,13 +195,6 @@ describe('Eth bridge test', () => {
       done();
     });
 
-    it('Queries generic asset balance for test token 1 for BridgeTests account', async done => {
```

Comment:
balance is verified at line https://github.com/cennznet/api.js/pull/365/files#diff-f1bb908fb5b8ebb0dc7b51bdde1abe493f8632c8981ca21836dbaff0a990e47eR66